bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,14 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% else %}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
```

Comment:
I think if we just update this comment with @flodolo's suggestion we're good to squash & merge.